src/test/compile-fail/lint-exceeding-bitshifts.rs

```diff
@@ -11,7 +11,7 @@
 #![deny(exceeding_bitshifts)]
 #![allow(unused_variables)]
 #![allow(dead_code)]
-#![feature(num_bits_bytes)]
+#![feature(num_bits_bytes, const_indexing)]
```

Comment:
The bitshift checks have been errors since before 1.0. I'll investigate if anything assumes they errored in check_const.